packages/react-sdk/src/Experiment.spec.tsx

```diff
@@ -78,35 +86,57 @@ describe('<OptimizelyExperiment>', () => {
       expect(optimizelyMock.onReady).toHaveBeenCalledWith({ timeout: 100 })
       // while it's waiting for onReady()
       expect(component.text()).toBe('')
-      resolver.resolve()
+      resolver.resolve({ sucess: true })
 
       await sleep()
```

Comment:
Kinda nitty, but this seems prone to flakiness to me :( Maybe there is a point in the component lifecycle we can hook into?

Reply:
Could you share more detail about what you had in mind? I don't see how component lifecycle plays into this test.

Reply:
I was just thinking that "sleeping" is flaky in general and was wondering if there is a way to actually wait for something concrete to be resolved via a promise. This doesn't seem to be an issue yet though since it's used throughout the test.

Reply:
We might be able to use the `onReady` Promise here (which we are resolving by calling `resolver.resolve`) instead of this `sleep` function. Will try this.